src/txmempool.h

```diff
@@ -105,7 +98,24 @@ class CTxMemPoolEntry
     uint64_t nSizeWithAncestors;
     CAmount nModFeesWithAncestors;
     int64_t nSigOpCostWithAncestors;
+    size_t nTxCost;            //!< ... and avoid recomputing tx cost (also used for GetTxSize())
+    size_t nModSize;           //!< ... and modified size for priority
+    size_t nUsageSize;         //!< ... and total memory usage
+public:
+    mutable size_t vTxHashesIdx; //!< Index in mempool's vTxHashes
+private:
+    unsigned int entryHeight;  //!< Chain height when entering the mempool
 
+    // Space that is typically used as padding. Commented out so that it doesn't cause extra memory use on different architectures.
+    // char PADDING_FREE_SPACE[2];
+
+    // Bool flags
+    struct CTxMemPoolEntryFlags {
+        int UNUSED_BITS : 6;
```

Comment:
@sipa are you sure of the alignment? On my system it is 8 bytes, which leaves no padding for the CTxMemPoolEntry. Perhaps you're on OSX where size_t is 4 bytes not 8 (I didn't look at OSX).